update-keycloak-user-group/README.md

```diff
@@ -0,0 +1,11 @@
+# Update Keycloak User Group
+This script is used to add users to the judgment group in Keycloak.
+
+## Running the script
+There are two environment variables
+* `ENVIRONMENT` - This is used to build the url for Keycloak so needs to be either, integration, staging or empty string.
```

Comment:
What about the prod environment?

Reply:
I've updated this to say you need to set SUB_DOMAIN instead of ENVIRONMENT.